code/modules/heavy_vehicle/mech_helpers.dm

```diff
@@ -46,6 +46,11 @@
 	H.toggled()
 	return TRUE
 
+/mob/living/heavy_vehicle/proc/toggle_power_remote()
+	var/atom/movable/screen/mecha/toggle/power_control/P = locate() in hud_elements
```

Comment:
This code is presuming an element is found, which isn't guaranteed as far as I'm aware